<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Title_and_Version">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Title - Version.Revision</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>521</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Add_Files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear_Files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Bad_Data">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark - Bad Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Confirm_Arrhythmia">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Arrhythmia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reject_Arrhythmia">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reject Arrhythmia</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_Time">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_y_max">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999997.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_y_min">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>490</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_x_window">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BeatDetection">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Beat Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Arrhythmia_Analysis">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrhythmia Analysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Quality_Scoring">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality Scoring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Edit_Settings">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Settings</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_auto_y">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_auto_y_min">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Next_Arrhythmia">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Prev_Arrythmia">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_First_Arrythmia">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>|&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Last_Arrhythmia">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;|</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Signals">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>29</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_graph"/>
   </widget>
   <widget class="QComboBox" name="comboBox_time_column">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Signals</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_x_min">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start_of_file">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>|&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_prev_window">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next_window">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_end_of_file">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>460</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;|</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Files"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionBeat_Detection"/>
    <addaction name="actionQuality_Scoring"/>
    <addaction name="actionArhythmia_Analysis"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Files">
   <property name="text">
    <string>Open Files</string>
   </property>
  </action>
  <action name="actionBeat_Detection">
   <property name="text">
    <string>Beat Detection</string>
   </property>
  </action>
  <action name="actionQuality_Scoring">
   <property name="text">
    <string>Quality Scoring</string>
   </property>
  </action>
  <action name="actionArhythmia_Analysis">
   <property name="text">
    <string>Arhythmia Analysis</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
